<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinAuth</class>
 <widget class="QWidget" name="WinAuth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinAuth</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>451</width>
     <height>421</height>
    </rect>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>47</red>
      <green>172</green>
      <blue>255</blue>
     </color>
    </brush>
   </property>
   <property name="foregroundBrush">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>47</red>
      <green>172</green>
      <blue>255</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLog">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>140</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogin">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPass">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pass</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePas">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>170</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="butLogin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butRegister">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNameProg">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>60</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Test Mes</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
